src/main/java/geekbrains/java2/ht2_1/Board.java

```diff
@@ -0,0 +1,57 @@
+package geekbrains.java2.ht2_1;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Board {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.